### PR TITLE
Set select value to browser lang if not set in accounts service

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8895

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/pages/account.vue [hunk 1]
```diff
@@ -335,11 +336,16 @@ export default defineComponent({
...
-            )
-          : unref(languageOptions)?.find((o) => o.default)
+
+        if (languageConfiguration) {
+          selectedLanguageValue.value = unref(languageOptions)?.find(
+            (lO) => lO.value === languageConfiguration.value?.listValue?.values?.[0]?.stringValue
+          )
+        } else {
+          selectedLanguageValue.value = unref(languageOptions).find(
+            (lO) => lO.value === navigator.language
```
Comment (JammingBen) [inline packages/web-runtime/src/pages/account.vue:346]: This is what I meant. Please no `navigator.language` - that's already handled when the translations are being initialized.

```suggestion
            (lO) => lO.value === langauge.defaultLanguage
```

Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue:346]: yes, saw that, but I even dont know if we take this solution at all, this ha also some downsides

Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue:346]: not language.defaultLanguage but language.current

Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue]: I don't think thats the way to go, I guess current will fallback to en if its a language we don't support, so we will have set English there istead of Please choose, idk

Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue]: ok checked with zh, current is fine

**Pull request merged**